### PR TITLE
[0.5.5] Allow Customizing http.tls

### PR DESCRIPTION
**Pull request by hfuss**
Signed-off-by: hfuss <haydenfuss@gmail.com>

### CODE REVIEW

**Comment by calbritt** [comment]
reviewed and approved

File: manifests/mtls-cert.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+apiVersion: cert-manager.io/v1
```
Comment (drewmarshburn) [inline manifests/mtls-cert.yaml:2]: Do we need to adjust the expiration for this cert?

Comment (hfuss) [reply manifests/mtls-cert.yaml:2]: Its an example cert used for integration testing, not included as part of the actual chart.

Comment (drewmarshburn) [reply manifests/mtls-cert.yaml:2]: Thanks

**Comment by drewmarshburn** [comment]
I have reviewed and approve.

**Pull request merged**